jobs/build-triggers/tendrl-build-triggers-pr-tests.yml

```diff
@@ -0,0 +1,173 @@
+          default: ""
+          description: "Coma separated list of additional repos."
+      - string:
+          name: ADDITIONAL_REPOS_USM_CLIENT
+          default: ""
+          description: "Coma separated list of additional repos."
+    builders:
+      - shell: |
+          #!/bin/bash -xe
+          echo
+          if [[ "${{ghprbPullId}}" ]]; then
+            echo "ghprbPullId: ${{ghprbPullId}}"
+            echo "ghprbCommentBody: ${{ghprbCommentBody}}"
+            echo "ghprbPullLongDescription: ${{ghprbPullLongDescription}}"
+            ADDITIONAL_REPOS="${{ADDITIONAL_REPOS}},http://artifacts.ci.centos.org/tendrl/pr/{package_prefix}{package_name}/pr${{ghprbPullId}}/{package_prefix}{package_name}-pr${{ghprbPullId}}.repo"
+            echo
+            echo "Parse ghprbCommentBody and ghprbPullLongDescription for dependent PRs."
+            # filter only lines containing "depends on:" string (and remove it)
+            # so it will iterate over list similar to this:
+            # tendrl-notifier/pr153 tendrl-commons/pr820 ...
+            for dependency in $(echo -e "${{ghprbCommentBody}}\n${{ghprbPullLongDescription}}" | \
+                grep -i "depends on:" | sed 's/depends on: //I'); do
+              package=${{dependency%/*}}
+              pr_id=${{dependency#*/}}
+              ADDITIONAL_REPOS="${{ADDITIONAL_REPOS}},http://artifacts.ci.centos.org/tendrl/pr/${{package}}/${{pr_id}}/${{package}}-${{pr_id}}.repo"
+            done
+            echo
+          fi
+
+          # Prepare env.sh file
+          echo > ${{WORKSPACE}}/env.sh
+          # use cop tendrl/tendrl (master) repo
+          echo "TENDRL_REPO_URL=http://copr-be.cloud.fedoraproject.org/results/tendrl/tendrl/epel-7-x86_64/" >> ${{WORKSPACE}}/env.sh
+          echo "TENDRL_REPO_GPGKEY_URL=https://copr-be.cloud.fedoraproject.org/results/tendrl/tendrl/pubkey.gpg" >> ${{WORKSPACE}}/env.sh
+          echo "TENDRL_DEPENDENCIES_REPO_URL=http://copr-be.cloud.fedoraproject.org/results/tendrl/dependencies/epel-7-x86_64/" >> ${{WORKSPACE}}/env.sh
+          echo "TENDRL_DEPENDENCIES_REPO_GPGKEY_URL=https://copr-be.cloud.fedoraproject.org/results/tendrl/dependencies/pubkey.gpg" >> ${{WORKSPACE}}/env.sh
+          # configure additional repos
+          echo "ADDITIONAL_REPOS=${{ADDITIONAL_REPOS}}" >> ${{WORKSPACE}}/env.sh
+          echo "ADDITIONAL_REPOS_USM_SERVER=${{ADDITIONAL_REPOS_USM_SERVER}}" >> ${{WORKSPACE}}/env.sh
+          echo "ADDITIONAL_REPOS_USM_NODES=${{ADDITIONAL_REPOS_USM_NODES}}" >> ${{WORKSPACE}}/env.sh
+          echo "ADDITIONAL_REPOS_USM_CLIENT=${{ADDITIONAL_REPOS_USM_CLIENT}}" >> ${{WORKSPACE}}/env.sh
+          set +x
+
+          echo
+          echo "##############################################################"
+          echo "Run whole test suite."
+          echo
+      - trigger-builds:
+          - project: "tendrl-2-cluster-gluster"
+            property-file: env.sh
+            block: true
+
+    publishers:
```

Comment:
Where the test results will be posted?

Reply:
The overall status of the installation and tests is reported back to the PR where it was triggered (as a `CentOS CI - tests` line in the *checks* section) and *Details* link points to the particular job.
More granular results are available in the subsequent jobs (see *Subproject Builds* section of the Jenkins job).

Additionally when it fails, it will notify me over email, so I can check if it is real installation or test failure, or if it is problem with the tests itself.